demos/flask/requirements.txt

```diff
@@ -0,0 +1,3 @@
+beaker>=1.13.0
+fhirclient>=4
+flask>=2.3.2
```

Comment:
Turns out, we didn't even directly use most of the flask app requirements - most of the previous requirements are actually deeper dependencies of flask & beaker (and not something we import ourselves). So... we don't need em here.